features/single-sign-on/README.md

```diff
@@ -5,6 +5,10 @@
 At the moment, this feature is available in all paid plans. But soon, it will be only available on custom enterprise pricing plans.
```

Comment:
Should we also create a page describing multi account login flow? I don't think we talk about it in our docs anywhere. This can also cover what happens in roll or reveal key operations.

Reply:
IMO the new login flow is pretty self evident to the user logging in ( all the instructions are provided on the screen  i.e. proper messages that email has been sent or enter OTP etc.) That's why there is no page for login. @nikniv let me know if necessary?

Reply:
We should add it. A small page/section with screenshots and simple explanations covering the flow should be enough.

Reply:
@aman051197 Send this in a new MR